optuna_dashboard/ts/types/index.d.ts

```diff
@@ -12,6 +12,7 @@ type TrialIntermediateValueNumber = number | "inf" | "-inf" | "nan"
 type TrialState = "Running" | "Complete" | "Pruned" | "Fail" | "Waiting"
 type TrialStateFinished = "Complete" | "Fail" | "Pruned"
 type StudyDirection = "maximize" | "minimize" | "not_set"
+type PreferenceFeedbackMode = "choose_worst" | "auto"
```

Comment:
How about removing "auto" since it's unsupported for now?

```suggestion
type PreferenceFeedbackMode = "choose_worst"
```